src/engine_mbedtls.c

```diff
@@ -117,11 +117,14 @@ tls_context *default_tls_context(const char *ca, size_t ca_len) {
 static void tls_debug_f(void *ctx, int level, const char *file, int line, const char *str);
 
 static void init_ssl_context(mbedtls_ssl_config *ssl_config, const char *cabuf, size_t cabuf_len) {
+#ifdef _WIN32
```

Comment:
why is this needed?

Reply:
It's not needed. I will remove.  It went in while I was tracking down a Windows crash, and I neglected to remove it when done.